### DISCUSSION TITLE
Sbt throwing error "java.lang.ClassCastException: class java.lang.UnsupportedOperationException cannot be cast to class xsbti.FullReload (java.lang.UnsupportedOperationException is in module java.base of loader 'bootstrap'; xsbti.FullReload is in unnamed module of loader 'app')"

### DISCUSSION THREAD

**Discussion by mhussain-1** [sbt/sbt#6911] [Q&A]
In build.properties file 
sbt.version=1.5.5

In build.sbt file
scala Version is 2.12.10

<img width="1098" alt="Screen Shot 2022-06-07 at 4 47 24 PM" src="https://user-images.githubusercontent.com/99477376/172371884-dba05578-8f2d-4bee-bb2c-41edb23191e2.png">

**Comment by SethTisue** [comment]
Have you tried upgrading to sbt 1.6.2?

**Reply by mhussain-1** [reply]
Yes, with sbt 1.6.2 it is working well. But with sbt 1.5.5 it is throwing an error.

**Comment by SethTisue** [comment]
okay. I suggest you upgrade and call it a day, then. you're highly unlikely to run into any compatibility issues with the upgrade. the 1.5.x branch is no longer maintained.

**Comment by guliverza** [comment]
I have the same exception
java.lang.ClassCastException: class java.lang.UnsupportedOperationException cannot be cast to class xsbti.FullReload (java.lang.UnsupportedOperationException is in module java.base of loader 'bootstrap'; xsbti.FullReload is in unnamed module of loader 'app')
with sbt version 1.10.6 and java 21.0.4

**Reply by eed3si9n** [reply]
Could you open an issue with repro steps please?
